src/test/java/Ping/DeadlineTest.java

```diff
@@ -0,0 +1,23 @@
+package Ping;
+
+import org.junit.jupiter.api.Test;
+import java.time.LocalDate;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+public class DeadlineTest {
+    @Test
+    public void todoCommand_ShouldHaveCorrectForamt() {

```

Comment:
```suggestion
    public void todoCommand_ShouldHaveCorrectFormat() {

```
Here I think there may be a spelling error, did you mean `Format` instead of `Foramt`?